### PR TITLE
add form to help

### PR DESCRIPTION
**Pull request by granatov**
add pair of pages with forms
Снимок экрана 2022-10-18 в 00 46 36
Снимок экрана 2022-10-18 в 00 46 41
Снимок экрана 2022-10-18 в 00 46 44

### CODE REVIEW

File: themes/blist/ukrainerInTrier/content/en/blog/need-help.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+date: 2021-07-15
+description: 'Guide to emoji usage in Hugo'
+tags: ['help']
+# thumbnail: https://picsum.photos/id/1050/400/250
+---
+
+<div class="form-help">
+    <form action="/action_page.php">
+  <label for="fname">First name:</label><br>
+  <input type="name" id="fname" name="fname" placeholder="Name"><br>
```
Comment (Likita) [inline themes/blist/ukrainerInTrier/content/en/blog/need-help.md:13]: Add styles from theme to form elements

File: themes/blist/ukrainerInTrier/content/en/blog/need-help.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  <label for="mail">Mail address:</label><br>
```
Comment (Likita) [inline themes/blist/ukrainerInTrier/content/en/blog/need-help.md:14]: Email

File: themes/blist/ukrainerInTrier/content/en/blog/need-help.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  <input type="mail" id="mail" name="mail" placeholder="example@mail.com"><br>
+  <label for="text">What help do you need?</label><br>
+  <input type="text" id="text" name="text" placeholder="I need ..."><br><br>
+  <input type="submit" value="Submit">
```
Comment (Likita) [inline themes/blist/ukrainerInTrier/content/en/blog/need-help.md:18]: <textarea> better

File: themes/blist/ukrainerInTrier/content/en/blog/need-help.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
+---
+author: 'Hugo Authors'
+title: 'I need help'
+date: 2021-07-15
```
Comment (Likita) [inline themes/blist/ukrainerInTrier/content/en/blog/need-help.md:4]: Form pages should not be like blog pages. They should be separate independent pages. No date or author.

File: themes/blist/ukrainerInTrier/content/en/blog/need-help.md (hunk 2 above)
Comment (granatov) [reply themes/blist/ukrainerInTrier/content/en/blog/need-help.md:14]: +

File: themes/blist/ukrainerInTrier/content/en/blog/need-help.md (hunk 3 above)
Comment (granatov) [reply themes/blist/ukrainerInTrier/content/en/blog/need-help.md:18]: +

File: themes/blist/ukrainerInTrier/content/en/blog/need-help.md (hunk 1 above)
Comment (granatov) [reply themes/blist/ukrainerInTrier/content/en/blog/need-help.md:13]: +

File: themes/blist/ukrainerInTrier/content/en/blog/need-help.md (hunk 4 above)
Comment (granatov) [reply themes/blist/ukrainerInTrier/content/en/blog/need-help.md:4]: +

**Review by granatov — COMMENTED** [review]
add styles to form (right colours to dark/white theme, add hover effects to submit)
change type of textarea
remove blogs stuff on the top of page
change title in social networks block.

**Comment by granatov** [comment]
Снимок экрана 2022-10-25 в 22 00 43
Снимок экрана 2022-10-25 в 22 00 47
Снимок экрана 2022-10-25 в 22 00 58

File: themes/blist/assets/css/styles.css [hunk 5]
```diff
@@ -1,10 +1,54 @@
...
+@import 'assets/css/site';
 /* Paginator */
-@import "assets/css/paginator";
+@import 'assets/css/paginator';
 /* Tailwind's utility classes - generated based on config file */
-@import "node_modules/tailwindcss/utilities";
\ No newline at end of file
+@import 'node_modules/tailwindcss/utilities';
+
+input,
```
Comment (Likita) [inline themes/blist/assets/css/styles.css:12]: This theme is using Tailwindcss lib. It's better to use classes from this library.
Here is an example.
https://tailwindui.com/components/application-ui/forms/input-groups

**Review by Likita — APPROVED** [review]

**Pull request merged**